From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: L1nar <dmscore@build.local>
Date: Sun, 14 Jun 2026 00:00:00 +0000
Subject: [PATCH] DMS Core - Add story-core.yml config


diff --git a/src/minecraft/java/net/minecraft/server/dedicated/DedicatedServer.java b/src/minecraft/java/net/minecraft/server/dedicated/DedicatedServer.java
--- a/src/minecraft/java/net/minecraft/server/dedicated/DedicatedServer.java
+++ b/src/minecraft/java/net/minecraft/server/dedicated/DedicatedServer.java
@@ -284,0 +285,8 @@
+        // DMS Core start - story-core.yml
+        try {
+            org.purpurmc.purpur.dmscore.DMSCoreConfig.init(new java.io.File("story-core.yml"));
+        } catch (Exception e) {
+            DedicatedServer.LOGGER.error("Unable to load DMS Core configuration", e);
+            return false;
+        }
+        // DMS Core end - story-core.yml
